### DISCUSSION TITLE
关于圆角的设置

### DISCUSSION THREAD

**Discussion by symbol23441** [ppoffice/hexo-theme-icarus#919] [Q&A]
有没有一个地方可以统一地改这些的圆角
![image](https://user-images.githubusercontent.com/74479485/130029806-d4aba567-f376-47ed-a0a0-84bcef47690f.png)
是icarus\include\style\base.styl中的$card-radius ?= $radius吗,但我没找到这个值在哪里设置
我是个小白,请多多指教

**Comment by symbol23441** [comment]
难道直接赋值?给它写死?

**Reply by ppoffice** [reply]
https://github.com/ppoffice/hexo-theme-icarus/blob/3174e966170000293b9ffc6b232842ec839b04e2/include/style/base.styl#L39

上面加一行`$radius = 123456px`

**Reply by symbol23441** [reply]
谢谢啦🙏
